pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.black]
+# https://github.com/psf/black
+line-length = 120
+exclude = "(.eggs|.git|.hg|.mypy_cache|.venv|_build|buck-out|build|dist)"
```

Comment:
I think the default exclude already cover this, isn't it?

Reply:
```suggestion
```